apps/gp2-server/src/data-providers/working-group.data-provider.ts

```diff
@@ -106,13 +115,8 @@ export function parseWorkingGroupToDataObject(
         membersList: gp2.WorkingGroupMember[],
         member: GraphQLWorkingGroupMember,
       ) => {
-        if (!isWorkingGroupMemberRole(member.role)) {
-          throw new Error(
-            `Invalid working group role on members : ${member.role}`,
-          );
-        }
         const user = member.user && member.user[0];
-        if (!user) {
+        if (!(user && member.role)) {
```

Comment:
i believe this is tested here right:
https://github.com/yldio/asap-hub/pull/2151/files#diff-b7cd257b868f623e71c6e1e34dab897113804b4842635229175e8df46b80287cR143
and what about a missing user, do you know?

Anywy i feel like this line could just be 
```suggestion
        if (!user || !member.role) {
```

Which to make makes more sense - it reads something like `if there is no user or no role`